spec/features/visitors/password_recovery_spec.rb

```diff
@@ -216,46 +216,55 @@ def reset_password_and_sign_back_in
   #   Given I do not remember my password as a user
   #   When I complete the form with invalid email addresses
   #   Then I receive a useful error
-  scenario 'user submits email address with invalid format' do
+  context 'user submits email address with invalid format' do
     invalid_addresses = [
       'user@domain-without-suffix',
       'Buy Medz 0nl!ne http://pharma342.onlinestore.com'
     ]
-    allow(ValidateEmail).to receive(:mx_valid?).and_return(false)
 
-    invalid_addresses.each do |email|
-      fill_in 'Email', with: email
-      click_button 'Send me reset password instructions'
+    it 'validates email field' do
+      allow(ValidateEmail).to receive(:mx_valid?).and_return(false)
 
-      expect(page).to have_content t('valid_email.validations.email.invalid')
+      invalid_addresses.each do |email|
+        fill_in 'Email', with: email
+        click_button 'Send me reset password instructions'
+
+        expect(page).to have_content t('valid_email.validations.email.invalid')
+      end
     end
   end
 
-  scenario 'user submits email address with invalid domain name' do
+  context 'user submits email address with invalid domain name' do
     invalid_addresses = [
       'foo@bar.com',
       'foo@example.com'
     ]
-    allow(ValidateEmail).to receive(:mx_valid?).and_return(false)
+    it 'validates email field' do
+      allow(ValidateEmail).to receive(:mx_valid?).and_return(false)
 
-    invalid_addresses.each do |email|
-      fill_in 'Email', with: email
-      click_button 'Send me reset password instructions'
+      invalid_addresses.each do |email|
+        fill_in 'Email', with: email
+        click_button 'Send me reset password instructions'
 
-      expect(page).to have_content t('valid_email.validations.email.invalid')
+        expect(page).to have_content t('valid_email.validations.email.invalid')
+      end
     end
   end
 
-  scenario 'user submits blank email address' do
-    click_button 'Send me reset password instructions'
+  context 'user submits blank email address' do
+    it 'validates email field' do
+      click_button 'Send me reset password instructions'
 
-    expect(page).to have_content t('valid_email.validations.email.invalid')
+      expect(page).to have_content t('valid_email.validations.email.invalid')
+    end
   end
 
-  scenario 'user submits blank email address and has JS turned on', js: true do
-    click_button 'Send me reset password instructions'
+  context 'user submits blank email address and has JS turned on' do
+    it 'validates required fields with JS', js: true do
+      click_button 'Send me reset password instructions'
 
-    expect(page).to have_content 'Please fill in all required fields'
+      expect(page).to have_content 'Please fill in all required fields'
+    end
   end
 
```

Comment:
What is the reasoning for moving from `scenario` -> `context`? According to http://stackoverflow.com/questions/11643747/rspec-describe-context-feature-scenario it sounds like it's best/a convention to use the feature/scenario combo since we're writing acceptance tests with capybara.